_ml-commons-plugin/index.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: default
+title: About ML Commons 
+nav_order: 1
+has_children: false
+has_toc: false
+---
+
+# ML Commons plugin
+
+ML Commons for OpenSearch eases the development of machine learning features by providing a set of common machine learning (ML) algorithms through transport and REST API calls. Those calls choose the right nodes and resources for each ML request and monitors ML tasks to ensure uptime. This allows you to leverage existing open-source ML algorithms and reduce the effort required to develop new ML features. 
+
+Models trained through the ML Commons plugin support two types of algorithms: 
+
+- Model-based algorithms such kmeans or Linear Regression. To get the best results, make sure you train your model first, then use the model to apply predictions. Linear Regression is only supported 
+- No-model based algorithm such as RCA. These algorithms can be executed directly through an `Executable` interface. 
```

Comment:
RCA not released yet, how about remove this part

Reply:
Is this documentation associated to a specific OpenSearch release version? If not, I think we can add a statement here that lists all the other algorithms (RCA, LogisticRegression, KNN, etc) that are in the planning to release soon, just to make ml-common looks more sufficient?

Reply:
This doc is for 1.3 release.

Reply:
I'll remove it for now. @Zhangxunmt, is there a way to see all of the algorithms we do support?